integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy

```diff
@@ -289,15 +289,15 @@ class LandingPageSpecification extends AbstractSpecification {
     driver.pageSource.contains('This individual was identified as needing a security review, as part of their categorisation, by Another User of LEEDS (HMP) on ' +  LocalDate.now().format('dd/MM/yyyy'))
 
     when: 'the security review page is only saved'
-    securityText << 'security info'
+    securityText << 'security info text'
```

Comment:
There are quite a few changes like this, when you are changing values in integration test, so it is less generic. 
As a matter of interest, what is the reason behind doing these?
Only asking, as the more files we change more likely we'll have conflicts with other ongoing work.
This PR has loads of refactoring.

Reply:
Based on a separate conversation in slack: "Integration test were failing due to minimum character count"

Reply:
as kornel said we dont need this change. unless we are checking for a specific scenario or specific text.
Technically we should not be checking any content changes through integration tests anyways.